client/my-sites/plan-features/index.jsx

```diff
@@ -262,6 +263,7 @@ class PlanFeatures extends Component {
 
 		return map( planProperties, properties => {
 			const {
+				available,
```

Comment:
As mentioned in a separate comment, it would be great if we could be consistent with this prop name throughout the plan features components.

Reply:
@tyxla I agree and this would make a good janitorial change. I don't want to add more things to this PR but I'll create another one